src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -80,14 +79,15 @@
     STRING_DIALOG_ADD_STRING("String dialog, add string", Localization.lang("String dialog, add string"), "ctrl N"),
     STRING_DIALOG_REMOVE_STRING("String dialog, remove string", Localization.lang("String dialog, remove string"), "shift DELETE"),
     SWITCH_PREVIEW_LAYOUT("Switch preview layout", Localization.lang("Switch preview layout"), "F9"),
-    SYNCHRONIZE_FILES("Synchronize files", Localization.lang("Synchronize files"), "ctrl F4"),
-    TOGGLE_ENTRY_PREVIEW("Toggle entry preview", Localization.lang("Toggle entry preview"), "ctrl F9"),
-    TOGGLE_GROUPS_INTERFACE("Toggle groups interface", Localization.lang("Toggle groups interface"), "ctrl shift G"),
+    SYNCHRONIZE_FILES("Synchronize files", Localization.lang("Synchronize files"), "ctrl shift F7"),
+    TOGGLE_ENTRY_PREVIEW("Toggle entry preview", Localization.lang("Toggle entry preview"), "alt 2"),
+    TOGGLE_GROUPS_INTERFACE("Toggle groups interface", Localization.lang("Toggle groups interface"), "alt 3"),
     UNABBREVIATE("Unabbreviate", Localization.lang("Unabbreviate"), "ctrl alt shift A"),
     UNDO("Undo", Localization.lang("Undo"), "ctrl Z"),
     UNMARK_ENTRIES("Unmark entries", Localization.lang("Unmark entries"), "ctrl shift M"),
-    WEB_SEARCH("Web search", Localization.lang("Web search"), "F5"),
-    WRITE_XMP("Write XMP", Localization.lang("Write XMP"), "ctrl F7"),;
+    WEB_SEARCH("Web search", Localization.lang("Web search"), "alt + 4"),
```

Comment:
We changed that so that all the sidepanes can be opened with `alt + Numberkey`, I think a blogpost is a good idea but I would wait untill the Github disucssion is over and eveeryone is in agreement.